src/Dns/Dns/help/Get-AzDnsRecordSet.md

```diff
@@ -128,7 +129,7 @@ This cmdlet then returns all record sets in the zone (of all names and types).
 Type: System.Nullable`1[Microsoft.Azure.Management.Dns.Models.RecordType]
 Parameter Sets: (All)
 Aliases:
-Accepted values: A, Aaaa, CAA, Cname, MX, NS, PTR, SOA, SRV, TXT, Tlsa, DS, Naptr
+Accepted values: A, Aaaa, CAA, Cname, MX, NS, PTR, SOA, SRV, TXT, Tlsa, DS, NAPTR
```

Comment:
Should be `Naptr` since its more than 3 characters and should match the case in the `RecordType` enum of the generated Sdk code

Reply:
Updated to Naptr to match `RecordType`